c/progress/progress_bar.cc

```diff
@@ -32,65 +32,71 @@ static void init_status_strings() {
 }
 
 
+progress_bar::~progress_bar() {}
 
-progress_bar::progress_bar() {
+
+void progress_bar_disabled::set_progress(double, double) noexcept {};
+void progress_bar_disabled::set_status_finished() {};
+void progress_bar_disabled::set_status_error(bool) {};
+void progress_bar_disabled::set_message(std::string&&) {};
+void progress_bar_disabled::refresh() {};
```

Comment:
semicolons not needed after function bodies

Reply:
Not sure how they got there :) Fixed.